docs/architecture/adr-061-liquid-staking.md

```diff
@@ -0,0 +1,81 @@
+
+These changes to the staking module have been in development for more than a year and have seen substantial industry adoption who plan to build staking UX. The internal economics at Informal team has also done a review of the impacts of these changes and this review led to the developement of the exempt delegation system. This system provides governance with a tunenable parameter for modulating the risks of pricipal agent problem called the exemption factor. 
+
+## Decision
+
+We implement the semi-fungible liquid staking system and exemption factor system within the cosmos sdk.
+
+A new governance parameter is introduced that defines the ratio of exempt to issued tokenized shares. This is called the exemption factor.
+
+Min self delegation is removed from the staking system with the expectation that it will be replaced by the exempt delegations sytem.
+
+When shares are tokenized, the underlying shares are transfered to a module account and rewards go to the module account for the TokenizedShareRecord. 
+
+There is no longer a mechanism to override the validators vote for TokenizedShares.
+
+
+### MsgTokenizeShares
+
+The MsgTokenizeShares message is used to create tokenize delegated tokens. This message can be executed by any delegator who has positive amount of delegation and after execution the specific amount of delegation disappear from the account and share tokens are provided. Share tokens are demoninated in the validator and record id of the underlying delegation.
+
+A user may tokenize some or all of their Delegation.
+
+They will recieved shares like cosmosvaloper1xxxx5 where 5 is the record id for the validator operator.
+
+A validator may tokenize their self bond but tokenizing more than their min self bond will be equivalent to unbonding their min self bond and cause the validator to be removed from the active set.
+
+MsgTokenizeSharesResponse provides the number of tokens generated and their denom.
+
+
+### MsgRedeemTokensforShares
+
+The MsgRedeemTokensforShares message is used to redeem the delegation from share tokens. This message can be executed by any user who owns share tokens and after execution the delegation appear for the user.
+
+### MsgTransferTokenizeShareRecord
+
+The MsgTransferTokenizeShareRecord message is used to transfer the ownership of rewards generated from the tokenized amount of delegation. The tokenize share record is created when a user tokenize his/her delegation and deleted and full amount of share tokens are redeemed.
```

Comment:
This just transfers rewards right? How is this different than setting a withdraw address? Or is this transferring the entire LS position?